Fixed problem with time usage bars reporting incorrect values

jrfarah opened the pull request:
The time usage bar at the top of the SNEx2 shows an incorrect fraction of time used depending on the size of the window:

![Screenshot 2024-03-25 at 3 57 47 PM](https://github.com/LCOGT/snex2/assets/14000301/d7ef75b8-26c9-4bed-9b02-d1c43f0c64a2)
![Screenshot 2024-03-25 at 3 58 29 PM](https://github.com/LCOGT/snex2/assets/14000301/9dc7e285-a5fb-4aed-a2e3-3b3a4982aaae)
_(these two views show the same usage but at different window scalings.)_

Implemented solution was to turn off scaling for the progress bar and make it a bit more economical space-wise. This will cut into some of the benefits of having an entirely dynamic top menu but works as a fix to ensure the time usage is reported correctly.

crpellegrino commented:
I'm not sure if this is working as intended on my local copy. To test, I set the 1m usage to 50% and the fractional semester length to 30%, but the bars look way too squished and the time usage part extends for way more than 50% of the total length. See attached screenshots. I'm not sure if this is a browser or screen size issue, but we may need to think of another fix.

![Screenshot from 2024-03-26 13-40-37](https://github.com/LCOGT/snex2/assets/42218592/d2906142-d2b7-4685-8f05-d4c00dbcd960)
 
![Screenshot from 2024-03-26 13-38-41](https://github.com/LCOGT/snex2/assets/42218592/0c6d7699-b1fc-467a-bacb-a383b791127f)

jrfarah commented:
Thanks for checking it out @crpellegrino. It looked fine on my browser, but this was something I was worried about. How did you set the 1m usage and fractional semester length? I want to try and reproduce it and implement a fix on my end.

jrfarah commented:
@crpellegrino Thanks for identifying the issue. I figured out what was causing the issue and implemented a fix. Here's a screenshot of the layout with your 1m usage and fractional semester length settings:

<img width="1511" alt="Screenshot 2024-04-01 at 10 11 51 PM" src="https://github.com/LCOGT/snex2/assets/14000301/c93059f5-b6c8-4152-b26c-3b9a2c65d0bc">

I also removed "Usage" which should help us win some space back. The time usage bars also no longer scale with window size, although the rest of the top bar does.

crpellegrino commented:
This looks good on my end
